.env.development

```diff
@@ -30,7 +30,8 @@ SHOW_MAINTENANCE_ALERT=''
 HOTJAR_APP_ID=''
 HOTJAR_VERSION=6
 HOTJAR_DEBUG=''
-ENABLE_SKILLS_QUIZ=true
+ENABLE_SKILLS_QUIZ='true'
 ENABLE_NOTICES=''
-FEATURE_ENABLE_PROGRAMS=false
+FEATURE_ENABLE_PROGRAMS=''
```

Comment:
`false` is actually being counted as a truthy value in this config setup. Changing to empty string so it is actually falsy.